sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -56,17 +56,32 @@ PRINT_CONFIG_MSG("USE_AIRSPEED_MS45XX automatically set to TRUE")
 #define MS45XX_I2C_DEV i2c2
 #endif
 
-/** Sensor I2C slave address (defaults 0x50, 0x6C and 0x8C) */
+/** Sensor I2C slave address (existing defaults 0x50, 0x6C and 0x8C)
+ */
 #ifndef MS45XX_I2C_ADDR
 #define MS45XX_I2C_ADDR 0x50
 #endif
 
-/** MS45xx pressure range in psi.
- * The sensor is available in 1, 2, 5, 15, 30, 50, 100, 150 psi ranges.
- * 1 psi max range should be ~100m/s max airspeed.
+/** MS45xx sensors pressure output type can be in PSI or InH2O, as defined in the datasheet
+ *  if not defined to be MS45XX_PRESSURE_OUTPUT_TYPE_InH2O then PSI is used
+ * */
+#ifndef MS45XX_PRESSURE_OUTPUT_TYPE_InH2O
+#define MS45XX_PRESSURE_OUTPUT_TYPE_InH2O 1
+#endif
+
+/** MS45xx pressure range in PSI or InH2O
+ * The sensor is available in many ranges, the datasheet of your pressure sensor will tell which one
+ * and what this range represents
  */
 #ifndef MS45XX_PRESSURE_RANGE
-#define MS45XX_PRESSURE_RANGE 1
+#define MS45XX_PRESSURE_RANGE 4
+#endif
+
+/* Pressure Type 0 = Differential, 1 = Gauge */
+#ifndef MS45XX_PRESSURE_TYPE
+#define MS45XX_PRESSURE_TYPE 0
+#else
+#define MS45XX_PRESSURE_TYPE 1 //Note there are theoretical more types than 2, e.g. Absolute, it is not implemented since rare we force it to 1
```

Comment:
if `MS45XX_PRESSURE_TYPE` is defined, then this line seems to re-define it

Reply:
Indeed since there is no >1 device I could find and scenaro to test or anyone else using that.